### PR TITLE
feat(integration): Add dbt cloud integration docs

### PR DESCRIPTION
**Pull request by lgodin-nr**
## Give us some context

* New Relic does not currently have a document on integrating dbt Cloud. This PR creates the documents for integrating dbt Cloud using the integration at https://github.com/newrelic-experimental/newrelic-dbt-cloud-integration
* This integration will have a quick start. The documents will be updated when the quickstart is complete. (The quickstart needs this document to be in production, so it will be an iterative process)

### CODE REVIEW

**Comment by akristen** [comment]
Hi @lgodin-nr ! Thanks for submitting this PR. This looks like it might be a good fit for a branching doc -- going to follow up with the team and see if we can submit it as a ticket for our upcoming sprint that starts on Monday.

Thanks a bunch for sending this!

File: src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+
+Our Dbt cloud integration with Airflow monitors the health of your dbt Cloud jobs
+and resources, helping you identify problems like failed runs, failed models and failed tests. 
+This integration runs on Apache Airflow and queries Snowflake for any failed tests if configured to do so.
+
+## Prerequisites
+* Dbt Cloud account with APIs enabled and using Snowflake as the database. 
+* Snowflake account where the dbt Cloud account runs on.
+* Existing Airflow environment >= 2.8.1 or ability to run docker compose.
+* New Relic account.
```
Comment (akristen) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx:20]: I notice this is in our Open Telemtry category. Are there any OTEL/new relic requirements? how does otel fit into this?

File: src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+
+This integration creates three custom events in NR1
+
+<CollapserGroup>
+<Collapser
+    id="collapser-1-in-group-2"
+    title="dbt_job_run"
+>
+
+**dbt_job_run:** Provides metadata and the status of all completed runs. This event does not include any data about models, snapshots, seeds and tests. Attributes include:
```
Comment (akristen) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx:170]: Is this an event data-type that the integration collects specific to this set up?

Comment (lgodin-nr) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx:174]: Yes, we use an Airflow DAG to create three custom event types in NR1.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx (hunk 1 above)
Comment (lgodin-nr) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx]: Oh shoot. No, this should not be OTEL. I'll move it.

File: src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx [hunk 3]
```diff
@@ -9,20 +9,24 @@ metaDescription: Dbt Cloud with Airflow and Snowflake integration
 freshnessValidatedDate: never
 ---
 
-Our Dbt cloud integration with Airflow monitors the health of your dbt Cloud jobs
+Our dbt Cloud integration with Airflow monitors the health of your dbt Cloud jobs
```
Comment (lgodin-nr) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx]: I'm OK with this change, but dbt Labs specifically spells dbt with a lowercase d. I only use an uppercase D when it starts a sentence. 

image

File: src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx (hunk 2 above)
Comment (lgodin-nr) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx]: To be more clear, the only data in this integration is the three custom events. It is different than typical APM integrations which focus on metrics and spans. In our case, we mostly care about discrete events (when something has failed, in particular)

**Comment by lgodin-nr** [comment]
Latest commit adds a short troubleshooting message

File: src/nav/infrastructure.yml [hunk 4]
```diff
@@ -176,6 +176,8 @@ pages:
             path: /docs/infrastructure/host-integrations/host-integrations-list/couchbase-monitoring-integration
           - title: Collectd integration
             path: /docs/infrastructure/host-integrations/host-integrations-list/collectd-integration
+          - title: dbt Cloud with Snowflake and Airflow Integration
```
Comment (lgodin-nr) [inline src/nav/infrastructure.yml:179]: This section is for integrations that use the infrastructure agent, but dbt Cloud is not using the infrastructure agent. Are we sure it belongs here?

File: src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,536 @@
...
+```
+
+</Step>
+<Step>
+Copy the contents of `airflow/dags` to the root of your Airflow dags folder 
+</Step>
+<Step>
+Create the five Airflow connections needed for the DAG. The following table provides the connection name the info to set it up:
+
+<table>
```
Comment (lgodin-nr) [inline src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx:62]: While a table makes sense in theory, this feels like too much text. Especially with the links that are broken up into multiple lines. This is difficult to read on a laptop, and will be more difficult on mobile.

File: src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,536 @@
...
+    run_status_humanized,
+    run_status_message
+from dbt_job_run
+since 7 days ago
+```
+</Collapser>
+
+<Collapser
+  id="collapser-1-in-group-2"
+  title="`dbt_resource_run`"
```
Comment (lgodin-nr) [inline src/content/docs/more-integrations/dbt-cloud/dbt-cloud-integration.mdx:354]: The backticks are showing in the preview. Was that intended? They look good inside the collapser.

**Review by lgodin-nr — COMMENTED** [review]
Thanks for all the great work on this @bradleycamacho . Left a few comments.

**Pull request merged**